nbconvert/templates/html/slides_reveal.tpl

```diff
@@ -211,13 +209,29 @@ require(
 
         Reveal.addEventListener('slidechanged', update);
 
-        var update_scroll = function(event){
-          $(".reveal").scrollTop(0);
-        };
-
-        Reveal.addEventListener('slidechanged', update_scroll);
+        function setScrollingSlide(scroll) {
+
+            if (scroll === "True") {
+              var h = $('.reveal').height() * 0.95;
+              var hpx = "" + h + "px";
+              $('section.present').find('section')
+                .filter(function() {
+                  return $(this).height() > h;
+                })
+                .css('height', hpx)
```

Comment:
Gah, CSS. It's probably doing the relative height relative to the wrong parent.

One more thing to try: `calc(100vh - 20px)`. vh units are 1% of the height of the viewport, so 100vh should be the full height available in the window.